app/services/organizations/create_service.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Organizations
+  class CreateService < BaseService
+    def initialize(params)
+      @params = params
+      super
+    end
+
+    def call
+      organization = Organization.new(params)
```

Comment:
would you avoid mass assignment here and list each attribute assigned to the organization like in the original code?